libs/hdf-converters/src/twistlock-mapper.ts

```diff
@@ -118,16 +120,24 @@ export class TwistlockMapper extends BaseConverter {
       }
     ],
     passthrough: {
-      raw: {
-        transformer: (
-          data: Record<string, unknown>
-        ): Record<string, unknown> => {
-          return data;
-        }
+      transformer: (data: Record<string, any>): Record<string, unknown> => {
+        const auxData = data;
```

Comment:
don't overwrite parameters like this if we don't need to

just do auxil_d: [{name: 'twistlock', data: _.omit(data.results[0], [blah])}]

Reply:
Oh I see what you mean.  The thing you need to keep in mind is that doing the parameter overwriting like we're doing here with the assignment occurring to an internal structure of the object is that the changes will propagate due to js being a pass reference by value type of language.  This is a really solid stackoverflow answer covering this with a nice example: https://stackoverflow.com/a/3638034/645647.  Ways to work around this are to make 'auxData' a full and proper clone or to manually specify the missing data in auxiliary_data which could probably be eased through judicious use of the spread operator like so:

```js
const a = {a: 5, b: [{c: 2, d: 4}]};
console.log({...a, b: _.omit(a.b[0], ['c'])});
```